tests/disabled_by_default.pyi

```diff
@@ -0,0 +1,9 @@
+# This test file checks that disabled-by-default error codes aren't triggered,
+# unless they're explicitly enabled
+from typing import (  # Y022 Use "tuple[Foo, Bar]" instead of "typing.Tuple[Foo, Bar]" (PEP 585 syntax)
```

Comment:
Not related to this PR, but would be nice if we emit this on the next line instead by associating it with the AST node for the alias.

Reply:
True, I can work on that separately!